### ISSUE TITLE
wandering bundle log spam

### ISSUE THREAD

**Issue by No1129** [NyanMC/ChromaGadgets#1]
it spams the log
i think its from me having 
[more bundles](https://www.curseforge.com/minecraft/mc-mods/more-bundles) loaded
could you add a ways to disenable the log spam or fix the bug

**Comment by NyanMC** [comment]
that information is useless to me unless you send me your log so i can see what message it is printing and which mod is printing it. you can upload a latest.log file using a service such as [mclo.gs](https://mclo.gs/), sending it here directly as text is strongly discouraged

**Comment by NyanMC** [comment]
Doing a bit more investigative work into that mod, actually, I believe I understand why this is happening. It appears to have its own bundle handling code which injects directly into BundleItem. The Wandering Bundle extends this class, however a BundleHandler is never set, causing it to fall back to the default. That mod prints warnings repeatedly if a bundle is using the default handler, with no way to disable those warnings (not even through mixin). This could be possible to fix on my end through direct compatibility, but is not a high priority.

tl;dr i'm pretty sure it's their fault and not mine

**Comment by No1129** [comment]
oh
